doctests/dt-string.js

```diff
@@ -0,0 +1,69 @@
+// EXAMPLE: string_tutorial
```

Comment:
Needs to match what's in the CLI example.

```suggestion
// EXAMPLE: set_tutorial
```